main.tf

```diff
@@ -565,7 +568,7 @@ resource "aws_subnet" "agentless_scan_public_subnet" {
 
 
   tags = {
-    Name           = "${var.prefix}-subnet"
+    Name           = "${var.prefix}-vpc"
```

Comment:
Everything else in the PR looks great, but I was confused by this change.

Reply:
➕ 1

Reply:
Yeah, we've gone back and forth on this. The CloudFormation had everything as "vpc" and this is actually a bit more accurate since the subnet belongs to the VPC. So this change makes things consistent.